.eslintignore

```diff
@@ -1,4 +1,5 @@
 dist-storybook-tmp/
+typography.ts
```

Comment:
Do we need this?

Reply:
Getting some eslint issues because of where that file sits in the directory structure. I think I need to pull it into `src` 🤔